apiary.apib

```diff
@@ -3014,44 +3319,180 @@ Restricted to *EDITOR* project role that has the permission to update metadata o
                 ]
             }
 
-### Get Dashboard by id [GET /rest/projects/{projectId}/md/dashboards/{id}]
-Returns Dashboard with given *id*.
+### Get Dashboard by id [GET /rest/projects/{projectId}/md/dashboards/{dashboardId}]
+
+Returns Dashboard identified by given *id*.
```

Comment:
V url se změnilo `id` na `dashboardId`, upravit ještě tady v této větě.

Reply:
měnit to `id` vs `typObjektuId` byla blbost, změním to zpět.